### PR TITLE
Removed unnecessary parameters & Added comments

### PR DESCRIPTION
**Pull request by yashLadha**
- Removed the unnecessary parameter passed in Resource function as the default is already set equal to that param only
- Added docstring for functions in the code

### CODE REVIEW

**Comment by yashLadha** [comment]
Review it @xeon-zolt @meets2tarun

**Comment by meets2tarun** [comment]
@yashLadha Please go through [README.md](https://github.com/fossasia/meilix-generator/blob/master/README.md) after the merge of the PR[#42](https://github.com/fossasia/meilix-generator/pull/42), in the meanwhile, you can see [this](https://github.com/fossasia/meilix-generator/pull/42/files).
Get yourself with the rules and contribution ways and get abide by it. 
Thanks

**Comment by yashLadha** [comment]
Should I create a separate Issue for it??? @meets2tarun

**Comment by meets2tarun** [comment]
@yashLadha It's ok for now.

**Comment by yashLadha** [comment]
Will make sure in future to abide it. :+1:

**Comment by yashLadha** [comment]
@meets2tarun Please review it.

File: app.py [hunk 1]
```diff
@@ -60,10 +61,9 @@ def inner():
 			for line in iter(proc.stdout.readline,''):
 				time.sleep(1)  # Don't need this just shows the text streaming
 				yield line.rstrip() + '<br/>\n'
-
 	else:
 		return redirect(url_for('index'))
-	return Response(inner(), mimetype='text/html')  # text/html is required for most browsers to show th$
+	return Response(inner())  # text/html is required for most browsers to show th$
```
Comment (abishekvashok) [inline app.py]: Then why was mimetype removed then?

Comment (yashLadha) [inline app.py]: Because in the documentation it is already set as default.

Comment (abishekvashok) [inline app.py]: You can follow the steps in the documentation,
But tune it accordingly :)

Comment (abishekvashok) [inline app.py]: You can follow the steps in the documentation,
But tune it accordingly :)

Comment (yashLadha) [inline app.py]: According to me, if it is set by default, them i felt that there is no need for an extra prameter to pass. What are your views @abishekvashok

Comment (abishekvashok) [inline app.py]: > text/html is required for most browsers to show this

Comment (yashLadha) [inline app.py]: But it is working perfectly fine, and not breaking up the code.

Comment (abishekvashok) [inline app.py]: can you assure it runs well, in old browsers also pls?

**Comment by yashLadha** [comment]
Yes, I myself tested it and by the way it is already implemented as
default, them why to pass it as the parameter

**Comment by abishekvashok** [comment]
Please check Travis

**Comment by yashLadha** [comment]
`No Rakefile found` . How to resolve this @abishekvashok . There is probably issue with the config file supplied.

**Comment by meets2tarun** [comment]
Travis checked is failed because at the time this PR had been made .travis.yml hadn't been merged into the repo.
So no issue at all.

**Pull request merged**